### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by TusharBhandirge**
Added sample card components for the component library.

live-link : https://card-component-2022.netlify.app/card%20component/card

### CODE REVIEW

File: Card Component/card.css [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  
+}
+
+.main{
+    display: flex;
+    flex-direction: column;
+    gap : 2rem;
+    justify-content: center;
+    align-items: center;
+    background-color: #686672;
```
Comment (msdeshmukh009) [inline Card Component/card.css]: You can create variables for the color.

**Review by msdeshmukh009 — COMMENTED** [review]
Hey tushar. I think you have missed the cards with dismiss and badge which are mentioned in PRD so please make those variants as well.

File: Card Component/card.css [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.card{
+
+    background-color: white;
+    border: none;
+    margin-left:0px ;
```
Comment (srishti-maurya) [inline Card Component/card.css]: you may mention only 0 instead of 0px

**Review by srishti-maurya — APPROVED** [review]
You may add padding around the content of the cards to make it much better.

File: Card Component/card.css (hunk 1 above)
Comment (TusharBhandirge) [reply Card Component/card.css]: Hi Mahesh,
Added card with dismiss and card with badge, thanks for pointing out, I missed those earlier.
Also added color variables and padding on card content.

image
image

File: Card Component/card.css (hunk 2 above)
Comment (TusharBhandirge) [reply Card Component/card.css]: Thanks for suggestion, I will change it to 0.

**Comment by TusharBhandirge** [comment]
> You may add padding around the content of the cards to make it much better.

Hi Srishti,
I have added padding as you suggested.
Thanks

**Pull request merged**